src/api/db/index.js

```diff
@@ -105,6 +111,23 @@ export default function initBookshelf(config) {
         promises = [...promises, ...morePromises];
       }
 
+      let schools = this.schools();
+      schoolNames = await School.fetchAll();
+      schoolNames = schoolNames.serialize().map(row => row.name);
+
+      for (let name of labelNamesToAdd) {
+        if (schoolNames.indexOf(name) > -1) {
+          schoolNamesToKeep.push(name);
+        }
+      }
+
+      let school_tags = await Promise.all(schoolNamesToKeep.map(tag_name => School.createOrSelect(tag_name)));
+      let schoolPromises = school_tags.map(async (tag) => {
+        await schools.attach(tag)
+      });
+
+      promises = [...promises, ...schoolPromises];
+
```

Comment:
`schools` table is a relational table for labels just like `likes` table is for posts... So labels will always be tables, but if it happens to match a school name then a relationship to that school is generated.

In the code first I fetch `schoolNames` which as per card would be inserted into db using CSV or other manual method. I use the schoolNames to verify if a label name matches a school's name. If it does the relationship is created.